elephantblog/management/commands/blogping.py

```diff
@@ -69,8 +69,8 @@ def handle_noargs(self, **options):
                     create_kwargs = {
                         'content_object': entry,
                         'weblogname': site.name,
-                        'weblogurl': site.domain + ':/' + PINGING_WEBLOG_URL,
-                        'changesurl': site.domain + ':/' + entry.get_absolute_url(),  # noqa
+                        'weblogurl': 'http://' + site.domain + PINGING_WEBLOG_URL,
```

Comment:
This does not make sense yet. PINGING_WEBLOG_URL can be an URL or a name. Can you fix this as well?